Entities/Ingame/Club.cs

```diff
@@ -48,13 +48,13 @@ public Club (int memoryAddress, ArraySegment<byte> originalBytes, IVersion versi
 
 		public string Name { 
 			get {
-				return PropertyInvoker.GetString (ClubOffsets.Name, 0, OriginalBytes, MemoryAddress, DatabaseMode);
+				return PropertyInvoker.GetString (ClubOffsets.Name, -1, OriginalBytes, MemoryAddress, DatabaseMode);
 			}
 		}
 
 		public string ShortName {
 			get {
-				return PropertyInvoker.GetString (ClubOffsets.ShortName, 0, OriginalBytes, MemoryAddress, DatabaseMode);
+				return PropertyInvoker.GetString (ClubOffsets.ShortName, -1, OriginalBytes, MemoryAddress, DatabaseMode);
 			}
 		}
 
```

Comment:
I don't understand.
```Firstname```, ```Lastname``` and ```Commonname``` always were inside an object and the actual pointer to the string for FM14 and FM15 has always been 0xC. This has not changed in the latest version.
The method signature for ReadString is
```csharp
public static string ReadString(int currentAddress, int? addBufferIndex, int offset, bool isRead) { }
```
As you can see the ```addBufferIndex``` parameter is ```nullable``` which means that if you don't want the method to follow an additional offset, you can pass a null (or zero):
```csharp
if (!isRead)
    currentAddress = ProcessManager.ReadInt32 (currentAddress);

if (addBufferIndex > 0)
    currentAddress = ProcessManager.ReadInt32 (currentAddress + (int)addBufferIndex);
```

I can't see how the changes you made to move common things to a Person object changed this fact. The ```Firstname``` and ```Lastname``` offsets are pointers to their respective objects, and the actual string is 0xC further down within that object.